src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/debugger/remote/RemoteDebuggerClient.java

```diff
@@ -43,27 +43,24 @@
 public class RemoteDebuggerClient extends DebuggerBase implements JVMDebugger {
   private RemoteDebugger remoteDebugger;
   private RemoteThreadFactory threadFactory;
-  private boolean unalignedAccessesOkay = false;
```

Comment:
After getting rid of the `readCInteger` and `readJLong` overrides below, `unalignedAccessesOkay` is no longer needed. The `DebuggerBase` superclass understands the alignment requirements.